README.md

```diff
@@ -69,15 +69,6 @@
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LIVE DEMO -->
-
-## 🚀 Live Demo <a name="live-demo"></a>
-
-- [No live link]()
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
```

Comment:
- [ **OPTIONAL** ] Good job fixing your broken `MIT` license link. I know the last reviewer mistakenly told you to remove the `Live Demo` section from your documentation. As you don't have the deployed link yet for your repo, I suggest you add the `Live Demo` sub-section again and add in its content some thing like`N/A`, `Coming soon`,... instead of removing it from your documentation. I you check from the **[Microverse README template](https://github.com/microverseinc/readme-template)** you will notice that it is one of the required sections to be present in your documentation. i.e.
```
## 🚀 Live Demo <a name="live-demo"></a>

- N/A or Not Available

<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

### Notice: 
Kindly make sure to use a same pull requesr and not merge it before it gets approved by a reviewer. Te whole milestone should use the same PR. You should apply this approach to all your next projects.